addons/mail/controllers/discuss/channel.py

```diff
@@ -74,7 +74,12 @@ def discuss_channel_members(self, channel_id, known_member_ids):
         channel = request.env["discuss.channel"].search([("id", "=", channel_id)])
         if not channel:
             raise NotFound()
-        return channel._load_more_members(known_member_ids)
```

Comment:
Note: (re)moving useless method called only once and which is just 1 line of code.